org.eclipse.xtext.testing/xtend-gen/org/eclipse/xtext/testing/AbstractLanguageServerTest.java

```diff
@@ -98,6 +100,20 @@
 import org.eclipse.xtext.ide.server.concurrent.RequestManager;
 import org.eclipse.xtext.resource.IResourceServiceProvider;
 import org.eclipse.xtext.service.OperationCanceledManager;
+import org.eclipse.xtext.testing.DefinitionTestConfiguration;
+import org.eclipse.xtext.testing.DocumentHighlightConfiguration;
+import org.eclipse.xtext.testing.DocumentSymbolConfiguraiton;
+import org.eclipse.xtext.testing.FileInfo;
+import org.eclipse.xtext.testing.FoldingConfiguration;
+import org.eclipse.xtext.testing.FormattingConfiguration;
+import org.eclipse.xtext.testing.HoverTestConfiguration;
+import org.eclipse.xtext.testing.RangeFormattingConfiguration;
+import org.eclipse.xtext.testing.ReferenceTestConfiguration;
+import org.eclipse.xtext.testing.SignatureHelpConfiguration;
+import org.eclipse.xtext.testing.TestCompletionConfiguration;
+import org.eclipse.xtext.testing.TextDocumentConfiguration;
+import org.eclipse.xtext.testing.TextDocumentPositionConfiguration;
+import org.eclipse.xtext.testing.WorkspaceSymbolConfiguration;
```

Comment:
These imports should not be necessary, since the imported classes reside in the same package as the class itself.

@Darkgaja Which Xtend version do you use? These imports should not be generated if you use Xtend 2.22 or above, see https://github.com/eclipse/xtext-xtend/issues/1014

Reply:
That's weird. I'm using Eclipse DSL 2020-12 with Xtend 2.24. Even when I'm deleting them manually, they get regenerated everytime I edit the Xtend file. However, using Gradle with the `-PcompileXtend=true` option generates this file correctly.

Reply:
the fix for import gen is in M1 and the build bootstraps against M2, so it has it. The 2.24 in your IDE wont have it.

Reply:
@miklossy the version is 2.25.0.M1 not 2.22.0